particles/Processing/LoadModel.particle

```diff
@@ -0,0 +1,12 @@
+import './schemas/MlResource.schema'
+import './schemas/MlModel.schema'
+
+particle LoadGraphModel in './js/LoadGraphModel.js'
+  in MlModel modelSpec
```

Comment:
Do you think it would be useful to rename `MlModel` to `MlModelReference` or `MlModelSpecification`? 
Further, do you think this is a good use case for inline schemas?

Reply:
I haven't seen all the PR, but I could see down-the-road that MlModel could refer to an in-memory model (or else, the actual model artifact) vs this, which is the location of the model.

Reply:
It should be named as for what it represents, and I'm not sure we have the domain knowledge yet. But 'MlModelSpecification' is ok. I think 'Reference' in 'MlModelReference' is too vague.

Using inline schemas is a more complicated issue then it appears, let's discuss in person?

Reply:
sure thing.